packages/spectacle/package.json

```diff
@@ -37,15 +37,12 @@
   "dependencies": {
     "@types/testing-library__jest-dom": "^5.14.5",
     "broadcast-channel": "^3.2.0",
-    "broadcastchannel-polyfill": "^1.0.0",
```

Comment:
Nice, looks like we now have this:

```ts
import { BroadcastChannel as BroadcastChannelPolyfill } from 'broadcast-channel';

// ...

const BroadcastChannel = window.BroadcastChannel || BroadcastChannelPolyfill;
```

and no longer using the `broadcastchannel-polyfill` lib